lib/services/database.dart

```diff
@@ -1,86 +1,89 @@
-import 'package:Covid19_PH/model/case_timeline.dart';
-import 'package:Covid19_PH/model/hospital_list.dart';
-import 'package:Covid19_PH/model/record_list.dart';
-import 'package:Covid19_PH/model/region_list.dart';
-import 'package:Covid19_PH/model/summary.dart';
+import 'package:Covid19_PH/model/base/jsonBase.dart';
 import 'package:meta/meta.dart';
 
 import 'package:Covid19_PH/services/api_path.dart';
 
 import 'network_helper.dart';
 
 class Database {
-  Future<Summary> getSummary() async {
+  JsonBase summary, caseTimeline, regionList, recordList, hospitalList;
```

Comment:
**This is prone to error.** 
Basically the consumer of this API can mistakenly pass `Summary` type to `CaseTimeLine`, `RegionList` type, and so on. 

change this **_from_**:
```JsonBase summary, caseTimeline, regionList, recordList, hospitalList;```

**_to_** their respective class types:
```
Summary summary;
CaseTimeLine caseTimeline;
RegionList regionList;
RecordList recordList;
Hospital hospitalList;
```